src/services/navigator/helpers.ts

```diff
@@ -14,6 +14,13 @@ import * as TypesLegacy from 'hyperview/src/types-legacy';
 import * as UrlService from 'hyperview/src/services/url';
 import { ANCHOR_ID_SEPARATOR } from './types';
 
+/**
+ * Dynamic and modal routes are not defined in the document
+ */
+export const isDynamicId = (id: string): boolean => {
```

Comment:
Does `dynamic` in the method name represents the same as the current navigator type "Dynamic"? I assume not given the code below, but naming also is confusing here.

Reply:
Agreed. In this case, I mean routes which are not pre-defined. Collectively, they are dynamically generated by the component. A better name for this and for the actual two screens would be ideal.